machine/v1beta1/types_awsprovider.go

```diff
@@ -44,6 +44,12 @@ type AWSMachineProviderConfig struct {
 	// it should use the default of its subnet.
 	// +optional
 	PublicIP *bool `json:"publicIp,omitempty"`
+	// NetworkInterfaceType specifies the type of network interface to be used for the primary
+	// network interface.
+	// Valid values are "interface", "efa", and omitted, which means no opinion and the platform
+	// chooses a good default which may change over time.
```

Comment:
good sentence about omitted, can you add

```
	// Valid values are "interface", "efa", and omitted, which means no opinion and the platform
	// chooses a good default which may change over time.
	// The current default is <blah>
```

Can I also get a link to the AWS doc in here.  I know it may go stale, but let's at least try.